react/src/components/Information.tsx

```diff
@@ -0,0 +1,325 @@
+    valid: boolean;
+    type: string;
+    licensee: string;
+    key: string;
+    expiration: string;
+  }>(
+    "licenseInfo",
+    () => {
+      return backendaiclient?.enterprise.getLicense();
+    },
+    {
+      // for to render even this fail query failed
+      suspense: false,
+    }
+  );
+
+  if (!licenseInfo) {
+    licenseInfo = {
+      valid: false,
+      type: t("information.CannotRead"),
+      licensee: t("information.CannotRead"),
+      key: t("information.CannotRead"),
+      expiration: t("information.CannotRead"),
+    };
+  }
+
+  const columnSetting: DescriptionsProps["column"] = {
+    xxl: 4,
+    xl: 4,
+    lg: 2,
+    md: 1,
+    sm: 1,
+    xs: 1,
+  };
+
+  return (
+    <Flex
+      direction="column"
+      align="stretch"
+      style={{ margin: token.marginSM, gap: token.margin }}
+    >
+      <Row gutter={[token.margin, token.margin]}>
+        <Col xs={24} xxl={12}>
+          <Card
+            style={{
+              height: "100%",
+            }}
+          >
+            <Descriptions
+              title={t("information.Core")}
+              bordered
+              column={columnSetting}
+            >
+              <Descriptions.Item
+                label={
+                  <DescriptionLabel title={t("information.ManagerVersion")} />
+                }
+              >
+                <Flex
+                  direction="column"
+                  style={{ gap: token.marginXXS }}
+                  align="start"
+                >
+                  Backend.AI {backendaiclient.managerVersion}
+                  <DoubleTag
+                    label={t("information.Installation")}
+                    value={backendaiclient.managerVersion}
+                  />
+                  {/* TODO: get manager_version_latest  */}
+                  {/* <DoubleTag
+                label={t("information.LatestRelease")}
+                value={"manager_version_latest"}
+              /> */}
+                </Flex>
+              </Descriptions.Item>
+              <Descriptions.Item
+                label={<DescriptionLabel title={t("information.APIVersion")} />}
+              >
+                {backendaiclient.apiVersion}
+              </Descriptions.Item>
+            </Descriptions>
+          </Card>
+        </Col>
+        <Col xs={24} xxl={12}>
+          <Card>
+            <Descriptions
+              title={t("information.Security")}
+              bordered
+              column={columnSetting}
+            >
+              <Descriptions.Item
+                label={
+                  <DescriptionLabel
+                    title={t("information.DefaultAdministratorAccountChanged")}
+                    subtitle={t(
+                      "information.DescDefaultAdministratorAccountChanged"
+                    )}
+                  />
+                }
+              >
+                {/* TODO: accountChanged  */}
+                {true ? (
```

Comment:
I have a question regarding "기본 관리자 게정 정보 변경". In the previous version, the information page always had this set as `true`. Additionally, I removed the `최신` tag for Manager Version also because it is empty.

I just followed the same values as the previous version of Information page. Please let me know if this is not correct.

Reply:
Hmm, I think this section(accountChanged) doesn't give useful information to user(superadmin).
It seems like a legacy part of user interface (I checked the code history according to the latest commit of `main` branch and found out that it hasn't been changed more than 3 years;).
Anyway. we need to change it and make it useful, as another issue.